app/models/course_user.rb

```diff
@@ -3,7 +3,7 @@ class CourseUser < ActiveRecord::Base
   after_initialize :set_defaults, if: :new_record?
   before_validation :set_defaults, if: :new_record?
 
-  enum role: { student: 0, teaching_assistant: 1, manager: 2, owner: 3 }
+  enum role: { student: 0, teaching_assistant: 1, manager: 2, owner: 3, auto_grader: 4 }
```

Comment:
So we will run our own system-level evaluator, we will create it when we set up Coursemology. If CS1101S wants their own evaluator, they can set up for their instance and give the instance level permission. If some other person wants their own evaluator, they can set up their own, sign up for an account, and just give that account auto_grader access